Changes.md

```diff
@@ -5,6 +5,15 @@ Features
 --------
 
 - PatternMatch : Added a new node for matching strings against wildcard patterns.
+- ColorChooser :
+  - Added sliders for TMI (temperature, magenta, intensity) color space.
+  - Added color field, a widget giving control of two channels of "RGB", "HSV" or "TMI" triplets. The third channel is held constant.
+  - The color field and RGB, HSV and TMI slider groups can now be toggled on or off.
+  - Default visibility of the UI elements can now be set at startup by setting `colorChooser:inlineOptions:` and `colorChooser:dialogueOptions:` entries for the inline chooser and dialogue chooser respectively, with the following suffixes :
```

Comment:
I think we should be referring to `colorChooser:inline:` and `colorChooser:dialogue:` prefixed metadata here?

Reply:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/6003/commits/19eb8a51a773e65d0ea3afc9271c6060bdfee1f0.